LICENSE

```diff
@@ -0,0 +1,201 @@
+  Apache License
```

Comment:
spaces are missing, `Apache License` should be centered, see https://raw.githubusercontent.com/hoodiehq/hoodie/master/LICENSE